### DISCUSSION TITLE
Renovate doesn't update packages using pnpm

### DISCUSSION THREAD

**Discussion by lx4r** [renovatebot/renovate#21529] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I'm trying to setup Renovate for a repo I created using SvelteKit with pnpm. 

**Wanted**: Renovate should create PRs to update packages both within the version range specified in the `package.json` and outside of it.

**Result**: The onboarding PR says "It looks like your repository dependencies are already up-to-date and no Pull Requests will be necessary right away." but this is the output of `pnpm outdated`:

```
┌────────────────────────────────────────┬─────────┬────────┐
│ Package                                │ Current │ Latest │
├────────────────────────────────────────┼─────────┼────────┤
│ @playwright/test (dev)                 │ 1.32.2  │ 1.32.3 │
├────────────────────────────────────────┼─────────┼────────┤
│ @sveltejs/adapter-static (dev)         │ 2.0.1   │ 2.0.2  │
├────────────────────────────────────────┼─────────┼────────┤
│ @sveltejs/kit (dev)                    │ 1.15.2  │ 1.15.5 │
├────────────────────────────────────────┼─────────┼────────┤
│ @typescript-eslint/eslint-plugin (dev) │ 5.57.1  │ 5.58.0 │
├────────────────────────────────────────┼─────────┼────────┤
│ @typescript-eslint/parser (dev)        │ 5.57.1  │ 5.58.0 │
├────────────────────────────────────────┼─────────┼────────┤
│ vitest (dev)                           │ 0.30.0  │ 0.30.1 │
└────────────────────────────────────────┴─────────┴────────┘
```

### What you tried so far.

I have modified the default config so that it now looks like this:

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base"
  ],
  "rangeStrategy": "update-lockfile"
}
```

### Relevant debug logs

Sorry, I don't know which part of the logs would be useful so [here's](https://gist.github.com/lx4r/baea0d33185a747f0dcd058cf32948a1) the full log from the dashboard.

Thanks in advance for the help :)

**Comment by viceice** [comment, marked as answer]
- #21480

that PR is required to support that range strategy

**Reply by lx4r** [reply]
Ah, thanks!
